lib/kitchen/directions/bake_notes/bake_numbered_notes/main.rb

```diff
@@ -11,8 +11,12 @@ def self.v2(book:, classes:)
         V2.new.bake(book: book, classes: classes)
       end
 
-      # Used by V1, V2
-      def self.bake_note_exercise(note:, exercise:, divider: ' ')
+      def self.v3(book:, classes:, suppress_solution: true)
```

Comment:
Like I think @KarinaMendez2 or someone said the other day, can you write docs inline here that say what is different about v3?